### PR TITLE
Hex output most keys

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Breezewish <breezewish@pingcap.com>

## What have you changed? (mandatory)

Use hex to print keys, in most of the places.

- [ ] Merge https://github.com/pingcap/kvproto/pull/409 first

## What are the type of the changes? (mandatory)

- Improvement (change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

Manually:

```
[2019/06/13 22:13:02.228 +08:00] [ERROR] [endpoint.rs:452] [error-response] [err="locked primary_lock: hex(7480000000000000055F698000000000000001012500000000000000F801726F6F7400000000FB) lock_version: 409058720359120902 key: hex(74800000000000000D5F6980000000000000010173716C5F6D6F6465FF0000000000000000F7) lock_ttl: 3047"]

[2019/06/13 22:14:04.210 +08:00] [INFO] [apply.rs:1603] ["split region"] [keys="key 7480000000000000FF2700000000000000F8"] [region="id: 2 start_key: hex(7480000000000000FF2500000000000000F8) region_epoch { conf_ver: 1 version: 18 } peers { id: 3 store_id: 1 }"] [peer_id=3] [region_id=2]

[2019/06/13 22:14:04.210 +08:00] [INFO] [apply.rs:1039] ["execute admin command"] [command="cmd_type: BatchSplit splits { requests { split_key: hex(7480000000000000FF2700000000000000F8) new_region_id: 38 new_peer_ids: 39 } right_derive: true }"] [index=50] [term=6] [peer_id=3] [region_id=2]

[2019/06/13 22:13:01.773 +08:00] [INFO] [peer.rs:320] ["on split"] [split_keys="key 7480000000000000FF1F00000000000000F8"] [peer_id=3] [region_id=2]
```

### CODE REVIEW

**Comment by breeswish** [comment]
@siddontang @overvenus @BusyJay PTAL

File: Cargo.toml [hunk 1]
```diff
@@ -136,7 +136,7 @@ codec = { path = "components/codec" }
 tipb_helper = { path = "components/tipb_helper" }
 tipb = { git = "https://github.com/pingcap/tipb.git" }
 # TODO: use master branch after the next version is released.
-kvproto = { git = "https://github.com/pingcap/kvproto.git", branch = "release-3.0" }
+kvproto = { git = "https://github.com/pingcap/kvproto.git", rev = "3989027cb1ea74a513205d84adbce69d7e46b97d" }
```
Comment (siddontang) [inline Cargo.toml]: em, can we still use release-3.0?

Comment (breeswish) [reply Cargo.toml]: Yes, as long as we merge https://github.com/pingcap/kvproto/pull/409 first.

**Comment by breeswish** [comment]
Closed temporary since it is in wrong branch.

**Pull request closed without merging**